src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -11,13 +11,17 @@
 using Newtonsoft.Json.Linq;
 using NuGet.Frameworks;
 using NuGet.LibraryModel;
+using NuGet.Packaging.Core;
 using NuGet.RuntimeModel;
 using NuGet.Versioning;
 
 namespace NuGet.ProjectModel
 {
     public class JsonPackageSpecReader
     {
+        public const string PackOptions = "packOptions";
```

Comment:
const will add the string into the referencing assembly, if the string changes it could then be incorrect. Typically in nuget we use static readonly for public, and const for non-public.